src/document_basic.c

```diff
@@ -10,6 +10,9 @@
 #include "module.h"
 #include "rmutil/rm_assert.h"
 
+#define MILLISECOND_IN_ONE_SECOND 1000
+#define NANOSECOND_IN_ONE_MILLISECOND 1000000
```

Comment:
Already defined in `spec.c`. Let's not have double definitions of the same macros